conu/apidefs/file.py

```diff
@@ -0,0 +1,84 @@
+"""
+Abstract class definitions for files in containers.
+"""
+
+from conu.apidefs.container import Container
+from conu.apidefs.image import Image
+
+
+class File(object):
+    """
+    File class definition which contains abstract methods. The instances should call the
+    constructor
+    """
+    def __init__(self, mount_dir):
+        """
+        :param mount_dir: str, directory for mounting container
+        """
+        self.container = None
+        self.container_id = None
+        self.image = None
+        self.image_id = None
+        self.mount_dir = mount_dir
+
+    def mount_container(self, container, container_id):
```

Comment:
can we add this method to Container class, let it accept `mount_point=None` and return used mount_point?